packages/backend-dynamic-feature-service/src/schema/openapi.yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.0.3
+info:
+  title: core.dynamicplugins.frontendRemotes
+  version: '1'
+  description: The Backstage backend plugin that serves the frontend plugins module federation manifests and assets
+  license:
+    name: Apache-2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: {}
+servers:
+  - url: /
+components:
+  examples: {}
+  headers: {}
+  parameters: {}
+  requestBodies: {}
+  responses:
+    ErrorResponse:
+      description: An error response from the backend.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+  schemas:
+    Error:
+      type: object
+      properties:
+        error:
+          type: object
+          properties:
+            name:
+              type: string
+            message:
+              type: string
+            stack:
+              type: string
+            code:
+              type: string
+          required:
+            - name
+            - message
+        request:
+          type: object
+          properties:
+            method:
+              type: string
+            url:
+              type: string
+          required:
+            - method
+            - url
+        response:
+          type: object
+          properties:
+            statusCode:
+              type: number
+          required:
+            - statusCode
+      required:
+        - error
+        - response
+      additionalProperties: {}
+  securitySchemes:
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+paths:
+  /manifests:
+    get:
+      operationId: GetManifests
+      description: Get the Module Federation manifest files of dynamic frontend plugins.
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties: {}
+                additionalProperties:
+                  type: string
```

Comment:
I'll assume this will need to be evolved later right? I would have expected something less "raw" that explains and names fields, like

```json
{
  "pluginManifests": [
    {
      "pluginId": "foo",
      ...
}
```

Reply:
I completed this schema to now return the real description of a Module Federation remote as expected by the Module Federation Runtime.
It's much more structured.